proto/process/agent.proto

```diff
@@ -29,6 +29,13 @@ message ResCollector {
 	CollectorStatus status = 3;
 }
 
+// CollectorProcHint is an enum that gives "hints" to backend services on how to handle
+// the parent message.
+enum CollectorProcHint {
+        hintUnspecified = 0;
```

Comment:
Do you need Unspecified? Because it's a `repeated` field, `Unspecified` would mean lack of a hint (empty set).

Reply:
Not necessary, but its the general practice.